File: src/main/java/me/taubsie/dungeonhub/application/exceptions/PlayerNotFoundException.java
package me.taubsie.dungeonhub.application.exceptions;

import me.taubsie.dungeonhub.kord.application.exceptions.CommandExecutionException;
import org.jetbrains.annotations.NotNull;

import java.util.UUID;

public class PlayerNotFoundException extends CommandExecutionException {

    public PlayerNotFoundException(@NotNull String name) {
        super("Player with name \"" + name + "\" not found!\n" +
                "\n" +
                "NOTE: If you think this is a mistake, try again in a few minutes.");
    }

    public PlayerNotFoundException(@NotNull UUID uuid) {
        super("Player with UUID \"" + uuid + "\" not found!");
    }
}

File: src/main/kotlin/me/taubsie/dungeonhub/application/exceptions/PlayerNotFoundException.kt
package me.taubsie.dungeonhub.application.exceptions

import java.util.*

class PlayerNotFoundException : CommandExecutionException {
    constructor(name: String) : super("Player with name \"$name\" not found!\n\nNOTE: If you think this is a mistake, try again in a few minutes.")

    constructor(uuid: UUID) : super("Player with UUID \"$uuid\" not found!")
}